docs/development.rst

```diff
@@ -491,3 +491,29 @@ value you can edit your ``.env`` file. For example::
 
 The ``docker-compose.yml`` will read this and start ``gunicorn`` and the
 ``celery`` worker with this setting.
+
+Configuring AWS S3
+==================
+
+The ``publish`` and ``unpublish`` Celery tasks and Django management commands
+require AWS S3 to be configured in order for them to do any real work, that is,
+creating/updating/deleting S3 objects used by the stage/production document API.
+In stage and production, the S3 bucket name as well as the AWS credentials are
+configured via the container environment, which in turn, gets the AWS credentials
+from a Kubernetes ``secrets`` resource. For local development, there is no need
+for any of this configuration. The ``publish`` and ``unpublish`` tasks will
+simply be skipped (although, for verification/debugging purposes, you can see
+the detailed skip messages in the ``worker`` log (
+``docker-compose logs -f worker``).
+
+However, if for testing purposes you'd like
+to locally configure the ``publish`` and ``unpublish`` tasks to use S3, you can
+simply add the following::
+
+    - MDN_API_S3_BUCKET_NAME=<your-s3-bucket-name>
+    - AWS_ACCESS_KEY_ID=<your-aws-access-key>
+    - AWS_SECRET_ACCESS_KEY=<your-aws-secret-key>
```

Comment:
Before I forget, r- on this.

Like we discussed, it should be possible to switch S3 publishing on and off from the .env in local dev. I don't think it affects Stage and Prod. Just a matter of local dev convenience. 
I think all of us core devs should always be publishing to S3 most days to help find bugs. 

(Just being verbose about this so we don't forget)

Reply:
With that said, *I* am using my *personal* AWS account. And it makes it easy that I don't see David's or Schalks file based on my MySQL on this laptop. 

How about we create some credentials and buckets for us guys and contributors can either A) use their personal AWS accounts or B) disable S3 publishing.